src/Microsoft.ML.TimeSeries/SRCNNAnomalyDetector.cs

```diff
@@ -95,15 +106,17 @@ public SrCnnArgument(SrCnnAnomalyDetector transform)
                 AvergingWindowSize = transform.InternalTransform.AvergingWindowSize;
                 JudgementWindowSize = transform.InternalTransform.JudgementWindowSize;
                 Threshold = transform.InternalTransform.AlertThreshold;
+                SrCnnDetectMode = transform.InternalTransform.SrCnnDetectMode;
+                Sensitivity = transform.InternalTransform.Sensitivity;
             }
         }
 
         private static VersionInfo GetVersionInfo()
         {
             return new VersionInfo(
                 modelSignature: "SRCNTRNS",
-                verWrittenCur: 0x00010001, // Initial
-                verReadableCur: 0x00010001,
+                verWrittenCur: 0x00010010, // Initial
```

Comment:
>Initial [](start = 46, length = 7)

Please change this to explain what caused the version change.

Reply:
Actually, where is the change in the way the model is serialized? I don't see any change to the serialization/deserialization methods in this file.

---
In reply to: [397223419](https://github.com/dotnet/machinelearning/pull/4946#discussion_r397223419) [](ancestors = 397223419)